run-network/creator.py

```diff
@@ -37,13 +37,17 @@ def create_env_cli(config_path: str):
 
 @cli.command()
 @click.argument("config_path")
-def start(config_path):
+@click.argument("profile", default="minimum", type=click.Choice(["minimum", "medium", "maximum"]))
+def start(config_path, profile):
```

Comment:
So in here, we would not need an extra argument:
```suggestion
def start(config_path):
```

The only downside I can think of is that the runtime choice cannot be type-checked by Click. (So instead, Compose would fail with a more cryptic error message.)

On the other hand, if the config file does get extended by additional profiles (in the future / by the user), then we would not interfere here (or have to keep the code in sync).